common/config/rush/browser-approved-packages.json

```diff
@@ -2,17 +2,61 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/approved-packages.schema.json",
   "packages": [
+    {
+      "name": "@fluentui/react",
+      "allowedCategories": [ "libraries" ]
+    },
+    {
+      "name": "@lifaon/path",
+      "allowedCategories": [ "libraries" ]
+    },
+    {
+      "name": "axios",
+      "allowedCategories": [ "libraries" ]
+    },
+    {
+      "name": "cors",
+      "allowedCategories": [ "libraries" ]
+    },
```

Comment:
This should go in `nonbrowser-approved-pacakges.json`

Reply:
moved